lib/commands/init.mjs

```diff
@@ -3,6 +3,19 @@ import fs from 'fs';
 import path from 'path';
 import ejs from 'ejs';
 import {getAbsolutePathFromRelativeToRoot} from "../utils/path_utils.mjs";
+import {isSystemDirectory} from "../commands/pathdetect.mjs";
+import { fileURLToPath } from 'url';
+import { dirname } from 'path';
+const __filename = fileURLToPath(import.meta.url);
```

Comment:
I think the variables are private because we are in a module, so no need to underscore them in the names.